kernel/fs/mod.rs

```diff
@@ -1,4 +1,5 @@
 pub mod devfs;
+pub mod ext2;
```

Comment:
Please implement ext2 as a library, in other words, a separate crate. How about moving it into `libs/ext2`?